mod/removeme.php

```diff
@@ -33,11 +33,12 @@ function removeme_post(App $a)
 
 	// send notification to admins so that they can clean um the backups
 	// send email to admins
-	$admin_mail_list = "'" . implode("','", array_map(dbesc, explode(",", str_replace(" ", "", $a->config['admin_email'])))) . "'";
-	$adminlist = q("SELECT uid, language, email FROM user WHERE email IN (%s)",
-		$admin_mail_list
-	);
-	foreach ($adminlist as $admin) {
+	$admin_mails = explode(",", str_replace(" ", "", $a->config['admin_email']));
+	foreach (admin_mails as $mail) {
```

Comment:
There is a `$` missing ahead of the `admin_mails` I think.

Reply:
Örgs